public/docs/ts/latest/tutorial/toh-pt1.jade

```diff
@@ -2,17 +2,18 @@ include ../_util-fns
 
 :marked
   # Once Upon a Time
+  
+  Every story starts somewhere. 
+  Our story starts where the [QuickStart](../quickstart.html) ends.
 
-  Every story starts somewhere. Our story starts where the [QuickStart](../quickstart.html) ends.
+  **Try it out**. Here's a link to a <live-example></live-example>.
```

Comment:
I'd probably strike "Try it out", though it's better because it's shorter and more to the point than the original. How about something like what's on the Architecture page?

"The code referenced on this page is available as a live example."